docs/source/migration_guide.rst

```diff
@@ -0,0 +1,123 @@
+.. contents::
+
+Migration Guide
+===============
+
+Version 2.4
+-----------
+
+Version 2.4 renames files and variables that included the string PAVICS.
```

Comment:
Nice docs.  Better here than buried in the PR.  If I remember correctly the old name starting with PAVICS are still supported for backward-compatibility, it's not mentionned here?

Reply:
Yes they are still supported (as this is a minor version update, not a major one). But we chose to not overly advertise this fact since we want to strongly encourage people to follow the migration guide (see example discussion here: https://github.com/bird-house/birdhouse-deploy/pull/428#discussion_r1513737235).

Backwards compatibility is described elsewhere: https://github.com/bird-house/birdhouse-deploy/blob/fd1a4b9cc29d89b2cf6205665ab387737955536c/CHANGES.md?plain=1#L153-L156 but I would like to keep the migration guide simply as a migration guide.